src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -3292,7 +3304,7 @@ public override IEnumerable<ISymbol> GetSymbolsWithName(Func<string, bool> predi
                 throw new ArgumentException(CSharpResources.NoNoneSearchCriteria, nameof(filter));
             }
 
-            return new PredicateSymbolSearcher(this, filter, predicate, cancellationToken).GetSymbolsWithName().GetPublicSymbols();
+            return new PredicateSymbolSearcher(this, filter, predicate, cancellationToken).GetSymbolsWithName().GetPublicSymbols()!;
```

Comment:
Why is this safe?

Reply:
`GetSymbolsWithName` returns an enumerable of non-null symbols. But there is not way to annotate `GetPublicSymbols` to say that it returns an enumerable of non-null symbols in this case (but it does).